static/openapi/vwrs-spec.json

```diff
@@ -1514,10 +1146,10 @@
       "patch": {
         "summary": "Update API key",
         "tags": [
-          "API Key"
+          "API key"
         ],
         "description": "Update the activation status of the API key.\n\nThis API requires _admin_ permissions to run.\n",
-        "operationId": "updateAPIKey",
+        "operationId": "updateAPI Key",
```

Comment:
sus

Reply:
Very. Let's see what @joubert-macrometa says.

Reply:
I think someone search-and-replaced "APIKey" with "API Key" and forgot to fix these.